WcaOnRails/app/views/teams/edit.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide(:title, "Editing team #{@team.friendly_id}") %>
+
+<div class="container">
+
+  <h3><%= yield(:title) %></h3>
+
+  <%= simple_form_for @team, html: { class: 'form-horizontal are-you-sure no-submit-on-enter' }, wrapper: :horizontal_form,
+    wrapper_mappings: {
+      check_boxes: :horizontal_radio_and_checkboxes,
+      radio_buttons: :horizontal_radio_and_checkboxes,
+      file: :horizontal_file_input,
+      boolean: :horizontal_boolean
+    }  do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+
+    <%= f.input :friendly_id, label: "Friendly ID" %>
+    <%= f.input :name %>
+    <%= f.input :description %>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        Members
+      </div>
+      <div class="panel-body form-inline">
+        <%= f.simple_fields_for :team_members do |t| %>
+          <div class="members <%= t.object.end_date != nil && t.object.end_date < Date.today ? 'past-member' : ''%>">
+            <%= t.input :user_id, required: true, as: :user_ids, only_one: true %>
+            <%= t.input :start_date, as: :date_picker, required: true %>
+            <%= t.input :end_date, as: :date_picker %>
+            <%= t.input :team_leader %>
+          </div>
+        <% end %>
+
+        <div class="links">
+          <%= link_to_add_association button_tag('Add member', type: "button", class: "btn btn-primary add"), f, :team_members %>
+        </div>
+      </div>
+    </div>
+
+    <div class="form-group">
+      <div class="col-sm-offset-2 col-sm-10">
+        <%= f.button :submit, class: "btn-primary" %>
+      </div>
+    </div>
+  <% end %>
+  <%= link_to 'Back to Teams', teams_path, class: "teams-home" %>
+
+</div>
+
+<script>
+//$(".add").click(function() {
+$(".panel-body").on('cocoon:after-insert', function() {
+  // activate datepicker and autocomplete on the new member
+  // currently not working...
```

Comment:
It works now.

On Mon, Mar 21, 2016 at 4:58 AM, Jeremy Fleischman <notifications@github.com
> wrote:

> In WcaOnRails/app/views/teams/edit.html.erb
> <https://github.com/cubing/worldcubeassociation.org/pull/390#discussion_r56789304>
> :
>
> > +
> > +    <div class="form-group">
> > +      <div class="col-sm-offset-2 col-sm-10">
> > +        <%= f.button :submit, class: "btn-primary" %>
> > +      </div>
> > +    </div>
> > +  <% end %>
> > +  <%= link_to 'Back to Teams', teams_path, class: "teams-home" %>
> > +
> > +</div>
> > +
> > +<script>
> > +//$(".add").click(function() {
> > +$(".panel-body").on('cocoon:after-insert', function() {
> > +  // activate datepicker and autocomplete on the new member
> > +  // currently not working...
>
> Is this still a problem?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/390/files/4cfc692de289d2d4d24e03be7e8db4b02da2a89d#r56789304>
>